### DISCUSSION TITLE
Cannot resize objects unless they are grouped first

### DISCUSSION THREAD

**Discussion by tardisx** [fabricjs/fabric.js#7536] [Q&A]
Hi all,

I raised a bug on this, but it's been marked as stale (https://github.com/fabricjs/fabric.js/issues/7485).

I don't doubt that it's a weird interaction between fabric and alpine rather than a fabric bug, but the test case is very simple and I'm not sure how to debug it any further. I'm hoping someone here has some insight as to what I might be doing wrong.

I think the fiddle makes the problem pretty clear: https://jsfiddle.net/gtkjmuqb/10/

Basically, the resize controls do not work, I can only move the object. But if I click+drag to select two of the objects, then single click to go back to selecting just one, the resize controls work!

Thanks in advance!

**Comment by asturur** [comment]
hi @tardisx your issue is good, is simply that we are having hard time to keep track of everything. I ll go ba to the issue
